src/components/pages/ReportPage.jsx

```diff
@@ -261,9 +263,8 @@ export default class ReportPage extends Component {
     const title = <FormattedMessage {...recordTypeConfig.messages.record.collectionName} />;
 
     let recordEditor;
-
     if (typeof normalizedCsid !== 'undefined' && normalizedCsid !== null) {
-      const isRunnable = canCreate(recordType, perms);
+      const isRunnable = canCreate(invocationType, perms) || canDelete(invocationType, perms);
```

Comment:
Remove canDelete. User only needs "C" perm to run. Just having "D" perm doesn't allow running. If a user has "D" perm, they _probably_ also have "C", but this code doesn't know or care about that.